README.md

```diff
@@ -98,6 +98,8 @@ Beautifier Options:
   -k, --keep-array-indentation      Preserve array indentation
   -x, --unescape-strings            Decode printable characters encoded in xNN notation
   -w, --wrap-line-length            Wrap lines at next opportunity after N characters [0]
+  -a, --wrap-attributes             Wrap attributes to new lines
```

Comment:
Just noticed that there is what seems to be an obsolete option using -W. I'll change the wrap-attributes to -A so there's no collision, since removing the other option shouldn't be done on this PR.